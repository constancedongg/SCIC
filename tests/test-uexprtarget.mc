/*
test derived unit
*/
|'{g} = 1000.0 '{kg}|;
float '{m*g} x;
float '{cm*kg} y;
int func main() {
    printl("in main");
    x = 9.98;
    y = x;
    printf(y);
    return 0;
}