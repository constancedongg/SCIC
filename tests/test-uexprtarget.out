in main
0.998